src/cameras/PerspectiveCamera.js

```diff
@@ -99,6 +107,39 @@ class PerspectiveCamera extends Camera {
 
 	}
 
+	/*
+	 * Computes 2D bounds of the camera's frustum at a given distance along the viewing direction.
+	 * Copies max/min height and width of the frustum's rectangle into minTarget and maxTarget.
+	 * Results are in the camera's local coordinate space, independent of its global position/rotation/scale.
+	 */
+	getFrustumBounds( distance, minTarget, maxTarget ) {
+
+		_tempV3.set( - 1, - 1, 0.5 ).applyMatrix4( this.projectionMatrixInverse );
+		_tempV3.multiplyScalar( distance / Math.abs( _tempV3.z ) );
+		minTarget.x = _tempV3.x;
+		minTarget.y = _tempV3.y;
+
+		_tempV3.set( 1, 1, 0.5 ).applyMatrix4( this.projectionMatrixInverse );
+		_tempV3.multiplyScalar( distance / Math.abs( _tempV3.z ) );
+		maxTarget.x = _tempV3.x;
+		maxTarget.y = _tempV3.y;
+
+	}
+
+	/*
+	 * Computes the height/width of the camera's frustum at a given distance along the viewing direction.
+	 * Copies the result into provided target Vector2 where x is width and y is height.
+ 	 */
+	getFrustumDimensions( distance, target ) {
```

Comment:
@Mugen87 I think for consistency with existing nomenclature, it should be `getFrustumSize()`.

Reply:
Sounds good!